src/renderer/js/component/shapeShift/internal/form.jsx

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import Link from "component/link";
+import { getExampleAddress } from "util/shape_shift";
+import { Submit } from "component/form";
+
+export default ({
+  values,
+  errors,
+  touched,
+  handleChange,
+  handleBlur,
+  handleSubmit,
+  resetForm,
+  isSubmitting,
+  shiftSupportedCoins,
+  originCoin,
+  originCoinDepositLimit,
+  updating,
+  getCoinStats,
+  receiveAddress,
+}) => {
+  return (
+    <form onSubmit={handleSubmit}>
+      <div className="form-field">
+        <span>{__("Exchange")} </span>
+        <select
+          className="form-field__input form-field__input-select"
+          name="originCoin"
+          onChange={e => {
+            getCoinStats(e.target.value);
+            handleChange(e);
+          }}
+        >
+          {shiftSupportedCoins.map(coin => (
+            <option key={coin} value={coin}>
+              {coin}
+            </option>
+          ))}
+        </select>
+        <span> {__("for LBC")}</span>
+        <p className="shapeshift__maxdeposit help">
+          {!updating &&
+            originCoinDepositLimit &&
+            `The most you can exchange is ${originCoinDepositLimit} ${
+              originCoin
+            }`}
+        </p>
+      </div>
+
+      <div>
+        <label>
+          {__("Return address")} ({__("optional but recommended")})
+        </label>
+        <input
+          type="text"
+          name="returnAddress"
+          placeholder={getExampleAddress(originCoin)}
+          onChange={handleChange}
+          onBlur={handleBlur}
+          value={values.returnAddress}
+          className="wunderbar__input wunderbar__input--shapeshift"
```

Comment:
Yeah definitely not a fan of the current input styling. I was thinking this was fine for now since the PR is getting bigger and bigger. But might as well do it now and avoid more tech debt. I will update the generic `input` style to be more inline with the wunderbar